### PR TITLE
Feature/ast 12643 sca panel

### PR DESCRIPTION
**Pull request by AndreGCX**
### Description

Improve SCA result details panel and add quick fix option when available.

### References

https://checkmarx.atlassian.net/browse/AST-9402

### Testing

UI tests for the new panel

### Checklist

- [x] I have added documentation for new/changed functionality in this PR (if applicable).
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used

### CODE REVIEW

File: src/main/resources/messages/CxBundle.properties [hunk 1]
```diff
@@ -33,6 +34,13 @@ INVALID_SCAN_ID=Invalid scan id
 LATEST_SCAN=latest
 SUMMARY=Summary
 DESCRIPTION=Description
+PATH=Vulnerable Package Paths
+ABOUT_VULNERABILITY=About this vulnerability
+FIND_PACKAGE_VERSION=Find best package version
+ADVISORY=Advisory
```
Comment (diogopcx) [inline src/main/resources/messages/CxBundle.properties:40]: Is ADVISORY used?

File: src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java [hunk 2]
```diff
@@ -141,11 +141,177 @@ public JPanel buildResultPanel(Runnable runnableDraw, Runnable runnableUpdater)
...
 
-        return JBUI.Panels.simplePanel(splitter);
+            return JBUI.Panels.simplePanel(splitter);
+        } else {
+            return buildScaPanel(result, runnableDraw, runnableUpdater);
+        }
+    }
+
+    @NotNull
+    private  JPanel buildScaPanel(Result result, @NotNull Runnable runnableDraw, Runnable runnableUpdater) {
```
Comment (diogopcx) [inline src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java:157]: We need to simplify this method.

File: src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java [hunk 3]
```diff
@@ -141,11 +141,177 @@ public JPanel buildResultPanel(Runnable runnableDraw, Runnable runnableUpdater)
                                                      DEFAULT_COLUMN);
         }
 
-        OnePixelSplitter splitter = new OnePixelSplitter();
-        splitter.setFirstComponent(PaneUtils.inVerticalScrollPane(details));
-        splitter.setSecondComponent(PaneUtils.inVerticalScrollPane(secondPanel));
+        if(!result.getType().equals(Constants.SCAN_TYPE_SCA)){
+            System.out.println("NOT SCA");
```
Comment (diogopcx) [inline src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java:145]: Is this something we should log or left over?

File: src/main/java/com/checkmarx/intellij/Constants.java [hunk 4]
```diff
@@ -29,7 +29,10 @@ private Constants() {
     public static final String FIELD_GAP_BOTTOM = "gapbottom 15";
 
     public static final String HTML_WRAPPER_FORMAT = "<html>%s</html>";
+    public static final String HTML_FONT_YELLOW_FORMAT = "<html><font color='#DCB33F'>%s</font></html>";
+    public static final String HTML_FONT_BLUE_FORMAT = "<html><font color='#2C7AE4'>%s</font></html>";
```
Comment (diogopcx) [inline src/main/java/com/checkmarx/intellij/Constants.java:33]: Do these constants work with different themes?

**Review by diogopcx — CHANGES_REQUESTED** [review]

**Review by pedrompflopes — APPROVED** [review]

**Pull request merged**